utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/source/KafkaAvroSourceFactory.scala

```diff
@@ -137,7 +138,7 @@ class KafkaAvroSourceFactory[K: ClassTag: NotNothing, V: ClassTag: NotNothing](v
     val formatterSchema = schemaBasedMessagesSerdeProvider.deserializationSchemaFactory.create[K, V](kafkaConfig, None, None)
     val recordFormatter = schemaBasedMessagesSerdeProvider.recordFormatterFactory.create[K, V](kafkaConfig, formatterSchema)
 
-    implProvider.createSource(params, dependencies, finalState.get, List(preparedTopic), kafkaConfig, deserializationSchema, recordFormatter, kafkaContextInitializer)
+    implProvider.createSource(params, dependencies, finalState.get, List(preparedTopic), kafkaConfig, deserializationSchema, recordFormatter, kafkaContextInitializer, KafkaTestParametersInfo.empty)
```

Comment:
Let's leave a comment that we don't support avro anymore so this feature won't be added here - use universal source instead